hybrid/decomposers.py

```diff
@@ -70,30 +72,144 @@ class EnergyImpactDecomposer(Runnable, traits.ProblemDecomposer):
             On unrolling reset condition, should :exc:`EndOfStream` be raised together
             with resetting/rewinding the subproblem generator?
 
+        traversal (str, optional, default='energy'):
+            Traversal algorithm used to pick a subproblem of `size`. Options are:
```

Comment:
Once you catch on it seems patently clear, but at first read it's actually not obvious that `size` is number of variables. I think it would help to spell this out in such first encounters: "pick a subproblem of `size` variables". In fact I think it still isn't 100% clear and would be helped if the next sentence was something like "The number of variables in the subproblem, `size`, ranges from 1 to the number of variables in the problem"